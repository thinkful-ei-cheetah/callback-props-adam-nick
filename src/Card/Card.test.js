import React from 'react'
import ReactDOM from 'react-dom'
import renderer from 'react-test-renderer'
import Card from './Card';

describe('Card Component', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');

    ReactDOM.render(<Card />, div);

    ReactDOM.unmountComponentAtNode(div);
  })

  it('renders the UI correctly', () => {
    const tree = renderer
      .create(<Card title="Test Card" content="This is a test card" />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  })
});