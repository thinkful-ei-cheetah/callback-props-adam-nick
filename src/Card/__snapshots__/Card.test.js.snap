// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card Component renders the UI correctly 1`] = `
<div
  className="Card"
>
  <button
    onClick={[Function]}
    type="button"
  >
    delete
  </button>
  <h3>
    Test Card
  </h3>
  <p>
    This is a test card
  </p>
</div>
`;
